tools/crm_mon.c

```diff
@@ -529,6 +529,12 @@ print_detail_cb(const gchar *option_name, const gchar *optarg, gpointer data, GE
     return TRUE;
 }
 
+static gboolean
+print_description_cb(const gchar *option_name, const gchar *optarg, gpointer data, GError **err) {
```

Comment:
brace on its own line

Reply:
The surrounding functions are incorrectly formatted. Should I make print_description_cb correct now, or should I leave it consistently incorrect and fix all of the functions in a separate PR?

Reply:
No need, we can fix them if we ever need to modify them